lib/phoenix_test/playwright/config.ex

```diff
@@ -115,11 +115,21 @@ schema =
       type: {:or, [:map, :keyword_list]},
       doc:
         "Define custom Playwright [selector engines](https://playwright.dev/docs/extensibility#custom-selector-engines)."
+    ],
+    sandbox_shutdown_delay: [
+      default: 0,
+      type: :non_neg_integer,
+      doc: """
+      Delay in milliseconds before shutting down the Ecto sandbox owner after a
+      test ends. Use this to allow LiveViews and other processes in your app
+      time to stop using database connections before the sandbox owner is
+      terminated. Default is 0 (immediate shutdown).
```

Comment:
This renders fine in ExDoc and is easier than using `<>`.

<img width="848" height="180" alt="Image" src="https://github.com/user-attachments/assets/0c1fde3a-0182-42cf-8779-d5b7a21dd8b4" />